packages/ember-runtime/tests/core/is_array_test.js

```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember-metal/core';
-import { isArray } from 'ember-runtime/utils';
 import ArrayProxy from 'ember-runtime/system/array_proxy';
 
 QUnit.module('Ember Type Checking');
 
 var global = this;
+var isArray = Ember.isArray;
```

Comment:
I spoke with @nathanhammond offline and asked whether it would be better to test `isArray` directly off of `Ember` or if I should `assert.equal(Ember.isArray, isArray)`, and he recommended the former. I can change this though.